### DISCUSSION TITLE
Scheduled sessions not showing up in /talks API results (review issue?)

### DISCUSSION THREAD

**Discussion by paulmelis** [pretalx/pretalx#1781] [Q&A] [answered]
I have manually added a number of sessions (due to our CfP being long closed), added external speakers, marked the sessions as Confirmed and gave them a spot in the schedule. However, those sessions refuse to show up in the `/talks` API results (either by ID, or words from the title), while other sessions appear fine. The only thing I can think of that's different is that they don't have any reviews. Could that have any influence on the results provided by the API?

![image](https://github.com/pretalx/pretalx/assets/1408287/96eab611-f8b5-4376-a97b-5cec4ac25eef)

![image](https://github.com/pretalx/pretalx/assets/1408287/7f677cfa-7d22-48b9-a9ba-5ba15a0fb277)

Edit: just noticed that for at least one of those missing sessions it also does not list a begin/end and room when using the Schedule > Export function to CSV.

**Comment by rixx** [comment, marked as answer]
Did you also publish a new schedule version after scheduling and confirming the talks? Do they show up in `/submissions`? `/talks` is just a filtered view of `/submissions`, so this is a matter of finding out which filter is keeping them out.

**Reply by paulmelis** [reply]
So submissions did list them. And it is indeed `/talks` appears to use the latest published schedule as basis, and not the latest internal version. Releasing a new schedule made the sessions show up.
